packages/frontend/src/router.definition.ts

```diff
@@ -121,10 +132,6 @@ const routes: RouteDef[] = [{
 		path: '/sounds',
 		name: 'sounds',
 		component: page(() => import('@/pages/settings/sounds.vue')),
-	}, {
-		path: '/accessibility',
-		name: 'accessibility',
-		component: page(() => import('@/pages/settings/accessibility.vue')),
 	}, {
```

Comment:
同様に

このsuggestionとpreferences.vueの`markerId`のsusggestionはペアです。

```suggestion
	}, {
		path: '/accessibility',
		redirect: '/settings/preferences#accessibility',
	}, {
```